lib/generators/rspec/scaffold/templates/edit_spec.rb

```diff
@@ -16,8 +16,12 @@
     assert_select "form[action=?][method=?]", <%= ns_file_name %>_path(@<%= ns_file_name %>), "post" do
 <% for attribute in output_attributes -%>
       <%- name = attribute.respond_to?(:column_name) ? attribute.column_name : attribute.name %>
+<% if Rails.version.to_f >= 5.1 -%>
```

Comment:
The problem here is that indentation here affects the output. We want the output to look 'normal' do we don't indent here.